content/docs/get-started/azure/modify-program.md

```diff
@@ -12,161 +12,235 @@ menu:
+@"
+<html>
+  <body>
+    <h1>Hello, Pulumi!</h1>
+  </body>
+</html>
+"@ | Out-File -FilePath site\index.html
 ```
 
 {{% /choosable %}}
+
+Now that you have your new `index.html` with some content, you can enable static website support, upload `index.html` to a storage container, and retrieve a public URL through the use of resource properties. These properties can be used to define dependencies between related resources or to retrieve property values for further processing.
+
+To see how this all works, open the program file in your IDE or text editor and add the following the lines right after the storage account creation.
+
+{{< chooser language "typescript,python,go,csharp" / >}}
+
 {{% choosable language typescript %}}
 
 ```typescript
-import * as pulumi from "@pulumi/pulumi";
-import * as azure from "@pulumi/azure";
+// Enable static website support
+const staticWebsite = new storage.StorageAccountStaticWebsite("staticWebsite", {
+    accountName: storageAccount.name,
+    resourceGroupName: resourceGroup.name,
+    indexDocument: "index.html",
+});
+```
 
-// Create an Azure Resource Group
-const resourceGroup = new azure.core.ResourceGroup("resourceGroup");
+{{% /choosable %}}
+{{% choosable language python %}}
 
-// Create an Azure resource (Storage Account)
-const account = new azure.storage.Account("storage", {
-    resourceGroupName: resourceGroup.name,
-    accountTier: "Standard",
-    accountReplicationType: "LRS",
-    tags: {
-        "Environment": "Dev",
-    },
+```python
+# Enable static website support
+static_website = storage.StorageAccountStaticWebsite("staticWebsite",
+    account_name=account.name,
+    resource_group_name=resource_group.name,
+    index_document="index.html")
+```
+
+{{% /choosable %}}
+{{% choosable language go %}}
+
+```go
+// Enable static website support
+staticWebsite, err := storage.NewStorageAccountStaticWebsite(ctx, "staticWebsite", &storage.StorageAccountStaticWebsiteArgs{
+    AccountName:       account.Name,
+    ResourceGroupName: resourceGroup.Name,
+    IndexDocument:     pulumi.String("index.html"),
+})
+if err != nil {
+    return err
+}
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+```csharp
+// Enable static website support
+var staticWebsite = new StorageAccountStaticWebsite("staticWebsite", new StorageAccountStaticWebsiteArgs
+{
+    AccountName = storageAccount.Name,
+    ResourceGroupName = resourceGroup.Name,
+    IndexDocument = "index.html",
 });
+```
+
+{{% /choosable %}}
+
+The static website resource leverages the storage account and resource group names defined previously in your program.
 
-// Export the connection string for the storage account
-export const connectionString = account.primaryConnectionString;
+Now use all of these cloud resources, and a local `FileAsset` resource, to upload `index.html` into your storage container.
+
+{{% choosable language typescript %}}
+
+```typescript
+// Upload the file
+["index.html"].map(name =>
```

Comment:
TypeScript seems to be the only language where you have this `map` from a single-element array. Is that by design?

Reply:
The TypeScript example was hand-authored rather than generated, and this was one difference. The version in the examples repo also included a 404 page, so the map doesn't make much sense here. I'd suggest just initializing the Blob directly.

Reply:
Yup, it was a copy-paste issue on my end. Now I'm following the structure of the other languages and just initializing the Blob directly.